test/regress/cli/regress1/bags/table_group3.smt2

```diff
@@ -0,0 +1,20 @@
+; DISABLE-TESTER: lfsc
```

Comment:
Can you comment on this, e.g. `; Disabled since table.group is not supported in LFSC`.